### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by csmir**
Grammar, reformatting & implements note in DNET issue #2029

### CODE REVIEW

**Comment by csmir** [comment]
Horrid commit history, can't resolve in any easy way unfortunately

**Review by quinchs — APPROVED** [review]

**Comment by quinchs** [comment]
Waiting on @lhjt

File: docs/guides/v2_v3_guide/v2_to_v3_guide.md [hunk 1]
```diff
@@ -62,24 +62,29 @@ GatewayIntents = GatewayIntents.AllUnprivileged | GatewayIntents.GuildMembers |
...
 
 The user is typing event has been changed to have both parameters cacheable.
 This allows you to download the user and channel if they aren't cached instead of them being null.
 
 ### Presence
 
-There is a new event called `PresenceUpdated` that is called when a user's presence changes,
-instead of `GuildMemberUpdated` or `UserUpdated`.
+There is a new event called ` PresenceUpdated ` that is called when a user's presence changes,
+instead of ` GuildMemberUpdated ` or ` UserUpdated `.
```
Comment (lhjt) [inline docs/guides/v2_v3_guide/v2_to_v3_guide.md]: > ` PresenceUpdated `

Is there any reason why you have decided to add an extra space between the graves and the term? 

This is inconsistent behaviour with: 

https://github.com/Discord-Net-Labs/Discord.Net-Labs/blob/074ed34215f314be800d96f52008a0268b86236d/docs/guides/v2_v3_guide/v2_to_v3_guide.md?plain=1#L16

and other files such as:

https://github.com/Discord-Net-Labs/Discord.Net-Labs/blob/074ed34215f314be800d96f52008a0268b86236d/docs/guides/voice/sending-voice.md?plain=1#L14

**Review by lhjt — CHANGES_REQUESTED** [review]

File: docs/guides/v2_v3_guide/v2_to_v3_guide.md (hunk 1 above)
Comment (csmir) [reply docs/guides/v2_v3_guide/v2_to_v3_guide.md]: Done subconsciously, thank you for noticing.

**Pull request merged**